scripts/update-factory-manifest

```diff
@@ -142,7 +142,13 @@ if [ -z "${remote_branch}" ]; then
 fi
 
 # fetch tags from upstream
-git fetch --tags --quiet ${foundries_manifest}
+if [ -z "${PREFIX}" ]; then
+    git fetch --tags --quiet ${foundries_manifest}
+else
+    # only fetch tags matching PREFIX
+    git ls-remote --tags ${foundries_manifest} | cut -f2 | grep 'refs/tags/${PREFIX}' | xargs git fetch origin
```

Comment:
I'm not super happy having to use `xargs` here .. if anyone has a better idea, let me know.

Reply:
Actually, I think there's a bug here.  `origin` (at the end) needs to be `${foundries_manifest}`

Reply:
Fixed.